src/api/resources/rest/user.ts

```diff
@@ -35,4 +35,55 @@ export class UserResource extends RESTResource<types.User> {
             return _.first(users);
         });
     }
+
+    /**
+     * Create a new (inactive) user account.
+     */
+    public create(user: types.CreateAccountInformation): Rx.Observable<types.User> {
+        return super.create(user);
+    }
+
+    /**
+     * Activate an inactive user account.
+     *
+     * @param token Activation token
+     */
+    public activateAccount(token: string) {
+        return this.callListMethod<types.ActivateAccountResponse>('activate_account', {token});
```

Comment:
Because `callMethod` is for a specific model instance (e.g. a specific user), but here you only have a token and you don't know the user (only the backend does).